src/soundbase.cpp

```diff
@@ -166,30 +166,38 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
```

Comment:
Lines 154 to 165 are irrelevant if this we enter this `if`.  They should be placed after the block.

Reply:
Might the problem be that this code is run once at startup and also during runtime if someone changes the sound devices?

Reply:
Before line 197 was introduced, the variable set there wasn't referenced until after this if block.  However, now line 197 _is_ there, it makes sense.